jet40.txt

```diff
@@ -0,0 +1,18 @@
+[Section]
+Name = Jet 4.0 Service Pack 8 (SP8) (KB829558) for 9x/NT
+Version = 4.0.7328
+License = Unknown
+Description = This installation includes Microsoft products that incorporate Microsoft Jet database engine version 4.0. 
+Size = 3.7 MiB
```

Comment:
Not needed since you specify the SizeBytes entry below.

Reply:
Sorry for both